.github/ISSUE_TEMPLATE/5-documentation-request.yml

```diff
@@ -0,0 +1,32 @@
+name: "📝 Documentation Request"
+description: "Request a change in documentation"
+title: "Docs: <a short description of my proposal>"
+labels:
+  - documentation
+  - triage
+body:
+  - type: checkboxes
+    id: sanity-checks
+    attributes:
+      label: Before You File a Bug Report Please Confirm You Have Done The Following...
```

Comment:
Not a "bug report", per se? Might be confusing (cf https://github.com/typescript-eslint/typescript-eslint/pull/4897#discussion_r866085661)